public/index.php

```diff
@@ -22,6 +22,6 @@
 // Default application
 $kernel = new CoreKernel(BASE_PATH);
 $app = new HTTPApplication($kernel);
-$app->addMiddleware(new ErrorControlChainMiddleware($app));
```

Comment:
@dnsl48 Why are you *replacing* error control chain here? Your middleware does something quite different, shouldn't it be added instead?

Reply:
we only have ErrorControlChainMiddleware in the first place here because we forgot to delete it from the recipe-core when we released 4.4

Reply:
lol fair enough.